components/sup/Cargo.toml

```diff
@@ -57,6 +61,9 @@ tabwriter = "*"
 tempdir = "*"
 time = "*"
 toml = { version = "*", default-features = false }
+tokio = "0.1"
+tokio-core = "0.1"
+tokio-io = "0.1"
```

Comment:
I locked these because it's recommended by the Tokio project. They make it clear that they'll be breaking things through each minor release until they 1.0. I'll lock to `= "*"` since that's what we're doing everywhere else.